Sources/DependencyInjectionPropertyWrapper.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  DependencyInjectionPropertyWrapper.swift
+//  Swinject
+//
+//  Created by Tyler Thompson on 12/17/19.
+//
+
+import Foundation
+@propertyWrapper
+public struct DependencyInjected<Value> {
+    let name:String?
```

Comment:
Please declare the properties private and add documentation for each of them. Thanks